### ISSUE TITLE
visceral organ asymmetry MP:0011248 - Review definition and placement

### ISSUE THREAD

**Issue by anna-anagnostop** [mgijax/mammalian-phenotype-ontology#3973]
visceral organ asymmetry [MP:0011248](https://www.informatics.jax.org/vocab/mp_ontology/MP:0011248) is currently defined as "lateral duplication of some or all internal organs" (with zero genotypes and zero annotations) and placed under heterotaxia MP:0004133.
Source: PMCID:[ PMC7443379](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7443379/)

The term label and definition don’t seem to go together; need to decide if placement is correct.
